packages/devtools_app/lib/src/inspector/inspector_controller.dart

```diff
@@ -107,6 +107,40 @@ class InspectorController implements InspectorServiceClient {
     }
   }
 
+  bool get debugSummaryLayoutEnabled => _debugLayoutSummaryEnabled.value;
+  final _debugLayoutSummaryEnabled = ValueNotifier(false);
```

Comment:
pass around the actual ValueNotifier object instead of adding these 4 methods that are all just accessing properties of 
debugLayoutSummaryEnabled so don't really add value but increase the size of the inspector_controller.dart api which is already large.

Reply:
can i keep the toggle method?

Reply:
sure that is reasonable to keep if you want